ts/adl/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "declaration": true,
+    "module": "ESNext",
+    "moduleResolution": "bundler",
+    "target": "ESNext",
+    "sourceMap": true,
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "isolatedModules": true
+  },
+  "include": ["src/**/*"]
+}
```

Comment:
For a small config file, tsconfig settings seem to conjure a world of pain. Perhaps move the common settings from 

```
ts/adl/tsconfig.json
ts/ui/tsconfig.json
```

into a common base?